src/main/java/com/example/account/elastic/repository/impl/AccountCriteriaBuilder.java

```diff
@@ -0,0 +1,45 @@
+package com.example.account.elastic.repository.impl;
+
+import org.springframework.data.elasticsearch.core.query.Criteria;
+
+public final class AccountCriteriaBuilder {
+
+    private final Criteria criteria = new Criteria();
```

Comment:
@amerkul don't hardcode your dependencies, it will be absolutely untestable

Reply:
Should I use a constructor with a parameter? Smth like 
```java
public AccountCriteriaBuilder(Criteria criteria) {
this.criteria = criteria;
}
```

Reply:
@amerkul yes